src/prefect/server/api/root.py

```diff
@@ -11,3 +14,21 @@
 async def hello():
     """Say hello!"""
     return "👋"
+
+
+@router.get("/ready")
```

Comment:
Technically we should have unit tests for this route. I'm okay with addressing separately though.

Reply:
grrr. yes this does need a test

Reply:
ok added - simple enough, lots of great examples in the repo already